README.md

```diff
@@ -187,7 +207,7 @@ function New-ResourceGroupRequirement {
         [string]$Location
     )
 
-    Push-Namespace "rg" {
+    New-RequirementGroup "rg" {
```

Comment:
I think this will error because it won't be able to resolve the parameter set.  We could change the implementation, but the easiest solution is to just use the snippet below.

```powershell
New-RequirementGroup "rg" -ScriptBlock {
    ...
}
```

Reply:
This works with Push-Namespace.  I've yet to try real world examples with New-RequirementGroup, but don't think it'd be different.

Reply:
Added for explicitness though

Reply:
oh then nvm